validator_client/src/duties_service.rs

```diff
@@ -674,84 +682,69 @@ async fn poll_beacon_attesters_for_epoch<T: SlotClock + 'static, E: EthSpec>(
         &[metrics::UPDATE_ATTESTERS_FETCH],
     );
 
-    let response = duties_service
-        .beacon_nodes
-        .first_success(
-            duties_service.require_synced,
-            OfflineOnFailure::Yes,
-            |beacon_node| async move {
-                let _timer = metrics::start_timer_vec(
-                    &metrics::DUTIES_SERVICE_TIMES,
-                    &[metrics::ATTESTER_DUTIES_HTTP_POST],
-                );
-                beacon_node
-                    .post_validator_duties_attester(epoch, local_indices)
-                    .await
-            },
-        )
-        .await
-        .map_err(|e| Error::FailedToDownloadAttesters(e.to_string()))?;
-
-    drop(fetch_timer);
-    let _store_timer = metrics::start_timer_vec(
-        &metrics::DUTIES_SERVICE_TIMES,
-        &[metrics::UPDATE_ATTESTERS_STORE],
-    );
+    // Request duties for either `INITIAL_DUTIES_QUERY_SIZE` validators or the count of validators for which we
+    // don't already know their duties for that epoch, whichever subset is larger. We use the `dependent_root`
```

Comment:
The first sentence here might need a slight tweak, as per recent changes.

Reply:
Updated, thanks!